CRM/Activity/BAO/Activity.php

```diff
@@ -2173,11 +2173,13 @@ public static function &exportableFields($name = 'Activity') {
             'title' => ts('Activity Type'),
             'name' => 'activity_type',
             'type' => CRM_Utils_Type::T_STRING,
+            'searchByLabel' => true,
```

Comment:
Hi - You'll see this failed the phpcs check - we use drupal coding standards - which specify TRUE rather than true (and 2 space indents)